util/logutil/log.go

```diff
@@ -110,9 +110,30 @@ func stringToLogLevel(level string) log.Level {
 	return defaultLogLevel
 }
 
+// logTypeToColor converts the Level to a color string.
+func logTypeToColor(level log.Level) string {
```

Comment:
How about change this to `func logTypeToColor(type string, level log.Level) string `
If type is not "highlight", return "\033".

Reply:
Only 'highlight` need to call this function and write color code.

Reply:
So why not call it logLevelToColor?